### PR TITLE
HTTP constants and error codes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Regilan** [comment]
@Vid-27 - Is there any issue with this pull request? I see it was closed. Reopening the branch

File: stash.py [hunk 1]
```diff
@@ -23,3 +24,78 @@ def connect_firestore_with_key(collection_name: str, firestore_key: str):
...
+    NOT_IMPLEMENTED = 501
+    BAD_GATEWAY = 502
+    SERVICE_UNAVAILABLE = 503
+    GATEWAY_TIMEOUT = 504
+    HTTP_VERSION_NOT_SUPPORTED = 505
+    VARIANT_ALSO_NEGOTIATES = 506
+    INSUFFICIENT_STORAGE = 507
+    LOOP_DETECTED = 508
+    NOT_EXTENDED = 510
+    NETWORK_AUTHENTICATION_REQUIRED = 511
```
Comment (Regilan) [inline stash.py]: @Vid-27 - Please add getter methods for this enum class. Refer to the below code.

    def status_code(self) -> int:
        return self.value[0]

    def status_message(self) -> str:
        return self.value[1]

File: stash.py [hunk 2]
```diff
@@ -23,3 +24,78 @@ def connect_firestore_with_key(collection_name: str, firestore_key: str):
         logger.info(f"{LOGGER_PREFIX}Firebase's firestore app is successfully connected")
         document_reference = database.collection(collection_name)
         return document_reference
+
+class HttpStatusCode(Enum):
+
+    # INFORMATIONAL RESPONSES (100–199)
+    CONTINUE = 100
```
Comment (Regilan) [inline stash.py]: Each parameter of this enum class should have a tuple value something like _CONTINUE = (100, 'Information Response - Continue')_ .

File: stash.py [hunk 3]
```diff
@@ -26,76 +26,82 @@ def connect_firestore_with_key(collection_name: str, firestore_key: str):
         return document_reference
 
 class HttpStatusCode(Enum):
+    def status_code(self) -> int:
```
Comment (Regilan) [inline stash.py:29]: @Vid-27 - Please have these methods at the end of the enum class

File: stash.py [hunk 4]
```diff
@@ -26,76 +26,82 @@ def connect_firestore_with_key(collection_name: str, firestore_key: str):
...
-    MULTIPLE_CHOICE = 300
-    MOVED_PERMANENTLY = 301
-    FOUND = 302
-    SEE_OTHER = 303
-    NOT_MODIFIED = 304
-    USE_PROXY = 305
-    UNUSED = 306
-    TEMPORARY_REDIRECT = 307
-    PERMANENT_REDIRECT = 308
+    MULTIPLE_CHOICE = (300,'Redirects -Multiple choice')
```
Comment (Regilan) [inline stash.py:54]: @Vid-27 - Kindly have spaces around the hyphen. Expecting it to be _'Redirects - Multiple Choice'_

**Comment by ghost** [comment]
Okay!

**Pull request merged**